pom.xml

```diff
@@ -398,16 +398,19 @@
 				<artifactId>commons-fileupload</artifactId>
 				<version>1.3.3</version>
 			</dependency>
+				<!-- https://mvnrepository.com/artifact/mysql/mysql-connector-java -->
```

Comment:
What is the use of the above comment?

Reply:
It was part of the comments in maven...I will remove them and push again